pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTaskRestletResource.java

```diff
@@ -213,6 +224,16 @@ public StringResultResponse getTaskStateDeprecated(
     return _pinotHelixTaskResourceManager.getTaskConfigs(taskName);
   }
 
+  @GET
+  @Path("/tasks/subtask/{taskName}/config")
+  @ApiOperation("Get the task state for the given task")
+  public Map<String, PinotTaskConfig> getSubTaskConfigs(
```

Comment:
(minor) We treat `subtask` as one word. Same for other places
```suggestion
  public Map<String, PinotTaskConfig> getSubtaskConfigs(
```